### PR TITLE
Refactor: CU-865cc1x8h: Change inline styles to tailwind classes

### PR DESCRIPTION
**Pull request by raulynzunza**
# Fixes ticket [#865cc1x8h](https://app.clickup.com/t/865cc1x8h)

## Problem Description

Change inline styles to tailwind classes on Hero and SearchBar components. Also, make unit test for each component.

## Checklist

- [x] Passes eslint linter
- [ ] Passes backend integration tests
- [x] Added tests for this change
- [ ] Added documentation for this change
- [ ] Removed unused packages

## Changes proposed in this pull request

Changed inline styles for tailwind classes, I also added some types of typing so we can have some options at the moment of creating a component.

## Reviewers

- @aviada-andreazayas 
- @jorgelabs 
- @aviada-rob 
- @AndresOL2001 

## Additional Comments

No additional comments.

### CODE REVIEW

**Comment by aviada-fgp** [comment]
Task linked: [CU-865cc1x8h Use tailwind classes and create unit tests](https://app.clickup.com/t/865cc1x8h)

**Review by aviada-andreazayas — APPROVED** [review]

File: __tests__/Molecules/hero.test.tsx
Comment (AndresOL2001) [inline __tests__/Molecules/hero.test.tsx:1]: everything looks good, only change the arbitrary values on the Hero Interface for tailwind closest values.

File: src/app/components/Molecules/Hero/Hero.tsx
Comment (AndresOL2001) [inline src/app/components/Molecules/Hero/Hero.tsx:1]: Change the albitrary value in the h1 className

File: src/app/components/Organisms/Navbar/Navbar.tsx
Comment (AndresOL2001) [inline src/app/components/Organisms/Navbar/Navbar.tsx:1]: Use the corresponding variable for the color#1067A8 in the border color and if is not there add the variable in the tailwind config file and use that variable.

File: src/app/dashboard/layout.tsx
Comment (AndresOL2001) [inline src/app/dashboard/layout.tsx:1]: Change albitrary values in the hero component

**Review by AndresOL2001 — CHANGES_REQUESTED** [review]

**Review by AndresOL2001 — APPROVED** [review]

**Pull request merged**